shared/src/main/scala/Rules.scala

```diff
@@ -0,0 +1,17 @@
+package carte
```

Comment:
Is it intended to use only `carte` as a package name?

Reply:
I call it `carte` for the moment. We can come up with a better name.

Reply:
It's removed now.